CHANGELOG.md

```diff
@@ -5,8 +5,21 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## x.y.z
 
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+## 7.0.0
+
 ### Added
 - Add support for PHP 8.2 (#1155)
+- Add official support for Symfony 6
```

Comment:
This was added in #1065 which was included in 6.0.0 so I don't think this changelog entry belongs here.

Reply:
Oh, right, I think we missed that. Going to change that back in a second.